README.md

```diff
@@ -1,4 +1,142 @@
-# Pop! Plymouth Theme
+# Plymouth Theme
 
-Plymouth splash screen for Pop!\_OS. For easy management of all Pop!\_OS related
-source code and assets, see the [main Pop! repo](https://github.com/pop-os/pop).
+Pop!_OS Plymouth theme using the ACPI BGRT graphics as background
+
+## Building and installing
+
+You'll need the following dependencies:
+* plymouth
+* plymouth-theme-spinner
+
+Verify that this theme is set as default:
+
+```bash
+$ sudo update-alternatives --config default.plymouth
+There are 2 choices for the alternative default.plymouth (providing /usr/share/plymouth/themes/default.plymouth).
+
+  Selection    Path                                                       Priority   Status
+------------------------------------------------------------
+  0            /usr/share/plymouth/themes/bgrt/bgrt.plymouth               110       auto mode
+  1            /usr/share/plymouth/themes/bgrt/bgrt.plymouth               110       manual mode
+* 2            /usr/share/plymouth/themes/pop-basic/pop-basic.plymouth   100       manual mode
+```
+
+Copy the latest theme
+
+```bash
+sudo cp -r pop-basic/* /usr/share/plymouth/themes/pop-basic/
+```
+
+and build a new initramfs to apply the changes
+
+```bash
+sudo update-initramfs -u -k all
+```
+
+If you want to test this is theme in your running X11 session, you'll need the following dependencies:
+* plymouth-x11
+
+## Testing
+
+⚠️ Plymouth draws over your desktop. Please note that you will need Terminal 2 on a different workspace to close Plymouth. ⚠️
+
+If you want to take screenshots, you should quit Wingpanel before
+
+```bash
+killall wingpanel
```

Comment:
`wingpanel` is used on Elementary OS, and isn't relevant on Pop. Otherwise I think this README is accurate.

Though the PR also removes `LICENSE.md` and `CODE_OF_CONDUCT.md`, I guess also from updating based on Elementary's theme. We probably want to still keep those assuming there isn't a specific reason to remove them (or the license is changed, in which case the file should be updated accordingly).